css/components/header/in-site/icon.css

```diff
@@ -0,0 +1,11 @@
+.header__in-site__icon {
+    height: 52px;
```

Comment:
use `block-size: 3.25rem` that should do it, only if your browser uses the browser default font size (16px)